prow/README.md

```diff
@@ -174,3 +174,7 @@ traffic to reach your hook and deck deployments.
  ```
 
 9. Add the webhook to GitHub.
+
+Hook processes the following events: issues, issue_comment, pull_request, push, status so github web hooks should be configured to send these event types.  It may be easier to configure the webhook to send everything.
```

Comment:
I suggest just sending everything, since we might add more event types in the future.